### PR TITLE
Add hardwareProfile to the bmhost

### LINKED ISSUE #158: missing 'hardwareProfile'
https://github.com/metal3-io/baremetal-operator/blob/b72e991c3409e44ca471bb4d27a02ac092c83248/cmd/make-bm-worker/main.go#L27

For baremetal hosts, it is required to have a 'hardwareProfile' in the spec section.

### PR DESCRIPTION
**Pull request by e-minguez**
Fixes #158

Please bear with me I have 0 golang experience

### CODE REVIEW

File: cmd/make-bm-worker/main.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ metadata:
   name: {{ .Name }}
 spec:
   online: true
+	hardwareProfile: {{ .HardwareProfile }}
```
Comment (elfosardo) [inline cmd/make-bm-worker/main.go]: fix indentation here

Comment (e-minguez) [reply cmd/make-bm-worker/main.go]: Fixed (I believe)

File: cmd/make-bm-worker/main.go [hunk 2]
```diff
@@ -81,11 +84,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *hardwareProfile == "" {
+		*hardwareProfile = "libvirt"
```
Comment (russellb) [inline cmd/make-bm-worker/main.go]: Let’s leave it empty if not specified and use the default

Comment (e-minguez) [reply cmd/make-bm-worker/main.go]: Fixed!

File: cmd/make-bm-worker/main.go [hunk 3]
```diff
@@ -26,6 +26,7 @@ metadata:
   name: {{ .Name }}
 spec:
   online: true
+  hardwareProfile: {{ .HardwareProfile }}
```
Comment (russellb) [inline cmd/make-bm-worker/main.go]: You can make this conditional where hardwareProfile doesn’t show up at all if it’s empty. See some examples in the dev-scripts version

**Pull request merged**